### PR TITLE
5934 create alert banner component

### LINKED ISSUE #5934: Create alert banner component
https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=26972-58415&t=DudVayaX18EkaNiq-11

Index - banner

### PR DESCRIPTION
**Pull request by bosiraphael**
Closes #5934

### CODE REVIEW

**Review by thomtrp — APPROVED** [review]

**Comment by Weiko** [comment]
@bosiraphael this is not used yet?

**Comment by lucasbordeau** [comment]
+1 for the story !

File: packages/twenty-front/src/modules/ui/layout/banner/components/BannerButton.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+const StyledBannerButton = styled.div`
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/ui/layout/banner/components/BannerButton.tsx:3]: Not clickable and text selectable.

Should we re-use a button we already have in our UI ?

**Review by lucasbordeau — CHANGES_REQUESTED** [review]
Button needs to be fixed

**Review by lucasbordeau — APPROVED** [review]
Used Button secondary instead

**Pull request merged**